src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt

```diff
@@ -38,19 +38,27 @@ class InitdbTest {
         val result = initdb.createDataSets( dbPath)
         println("result = $result")
         // To verify, we  check that the outputDir contains the expected files
-        val expectedFiles = listOf("gvcf_dataset", "hvcf_dataset")
-        expectedFiles.forEach { assert(File(dbPath + it).exists()) }
+        val expectedFiles = listOf("${dbPath}gvcf_dataset", "${dbPath}hvcf_dataset", "${dbPath}temp/tiledb_gvcf_createURI_error.log", "${dbPath}temp/tiledb_gvcf_createURI_output.log", "${dbPath}temp/tiledb_hvcf_createURI_error.log", "${dbPath}temp/tiledb_hvcf_createURI_output.log")
+        expectedFiles.forEach { assert(File( it).exists()) }
+
+        // Delete the log files created by the ProcessBuilder commands.
+        // Rerun the command .  The log files should NOT be created as
+        // the code will realize the dataset already exists and does not try to recreate it.
+        val expectedLogFiles = listOf("$dbPath}temp/tiledb_gvcf_createURI_output.log", "$dbPath}temp/tiledb_hvcf_createURI_error.log")
```

Comment:
Looks like you have unmatched "}"'s here. That is probably not what you want.

Reply:
@pjbradbury Thank you for catching this!  I fixed the test case then neglected to run it once more.  I have made the change and tested.